bin/eds-import-from-figma-api.js

```diff
@@ -167,14 +191,15 @@ const { identity } = require('lodash');
   // now, load in the local theme file. We want to look up keys in there
   const localTheme = jsonfile.readFileSync(filePath);
 
+  // Main loop: work thru each detected variable, and convert
   figmaApiReader
-    .getVariablesByCollectionId(edsCollection.id)
+    .getVariablesByCollectionId(tier2Collection.id)
     .forEach((figmaVariable) => {
       stats.total.push(figmaVariable.id);
       const variable = new FigmaVariable(
         figmaVariable,
-        collectionNameResponses.tier1Collection,
```

Comment:
The heart of the bug. I clearly documented this should be the tier 1 mode in the `FigmaVariable` API, but somewhere along the line forgot that the collection ID and the mode ID aren't synonymous

Future refactor should factor the main loop into a separate call, so that inputs to that loop can be tested (and the CLI for the script is separated out as its own thing)